frontend/app/services/lookup-service.server.ts

```diff
@@ -0,0 +1,65 @@
+import { z } from 'zod';
+
+import { getLogger } from '~/utils/logging.server';
+
+const logger = getLogger('user-service.server');
+
+export const preferredLanguageSchema = z.object({
+  id: z.string().optional(),
+  nameEn: z.string().optional(),
+  nameFr: z.string().optional(),
+});
+
+export type PreferredLanguageInfo = z.infer<typeof preferredLanguageSchema>;
+
+export interface LookupServiceDependencies {
+  readonly env: {
+    readonly INTEROP_API_BASE_URI: string;
+  };
+}
+
+export function getLookupService({ env }: LookupServiceDependencies) {
```

Comment:
@gregory-j-baker changed the service pattern and don't requires to pass service dependencies as argument. He also return the service as singleton at the end of the file. Update this file to match this new pattern.

https://github.com/DTS-STN/canadian-dental-care-plan/blob/main/frontend/app/services/user-service.server.ts

Reply:
Fixed: 72170e807b3f47c361f34e4bd4d9036444475779